testdata/p4_16_samples/issue3238.p4

```diff
@@ -1,17 +1,14 @@
-bit func(bit t1)
+bit func(int t1)
```

Comment:
why modify this test?

Reply:
This test is incorrect from the perspective of the (current) specification (a tuple cannot hold `int`). I tried to modify the test so that it tests the original intended behavior (implicit cast `int` -> `bit`, #3469).